AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -879,15 +881,13 @@ function Request-AzPowerShellModule {
 
     $storageModule = Get-Module -Name Az.Storage -ListAvailable | `
         Where-Object { 
-            $_.Version -eq [Version]::new(1,8,2) -or 
-            $_.Version -eq [Version]::new(1,11,1) 
-        } | `
-        Sort-Object -Property Version -Descending
+            $_.Version -eq [Version]::new(2,0,0) 

```

Comment:
greater than or equal to - the feature will be supported in future Az.Storage modules too.